source/daplink/drag-n-drop/flash_manager.c

```diff
@@ -95,6 +95,10 @@ error_t flash_manager_init(const flash_intf_t *flash_intf)
         return status;
     }
 
+#ifdef FLASH_PARTIAL_ERASE
+    // Processing for low-speed devices of chip erasure.
+    flash_manager_printf("    skip intf->erase_chip\r\n");
+#else
```

Comment:
I think the "flash_manager_init()" is preferred.
In the case of "flash_manager_mode()", it might appear to be able to change at any time.